stories/topics/con-gcp-upgrade-upgrading.adoc

```diff
@@ -0,0 +1,8 @@
+[id="con-gcp-upgrade-upgrading"]
+
+= Upgrading {PlatformNameShort}
+
+include::proc-gcp-run-upgrade-playbook.adoc[leveloffset=+1]
```

Comment:
With this include here you end up with two headings one after the other. There ought to be a line between. Even if its 'The following procedures form the upgrading process for {AAPonGCP}'